web/src/main/ui/app/components/flows-new/manage-flows/ui/step-icons-ui.component.scss

```diff
@@ -1,14 +1,18 @@
 .step-icons {
-  position: absolute;
+  //position: absolute;
```

Comment:
You can remove the position, top, and left css parameters since they are not used.

Reply:
Ok, I am gonna do it tmr.